src/main/java/org/jboss/forge/arquillian/api/ArquillianConfig.java

```diff
@@ -42,6 +43,14 @@ public boolean addExtensionProperty(String qualifier, String key, String value)
         return true;
     }
 
+    public boolean addExtensionProperty(String qualifier, Map<String, String> map) {
+        final Node node = xml.getOrCreate("extension@qualifier=" + qualifier);
```

Comment:
We have a lot of duplication with `extension@qualifier=`, `property@name=` - can we make constants out of them? Would it make it simpler?